### DISCUSSION TITLE
How can i get only videos link?

### DISCUSSION THREAD

**Discussion by devbusca** [Serene-Arc/bulk-downloader-for-reddit#926] [Q&A] [answered]
For example, only get the videos link from subreddit 'all', but obviously only from videos, not post or anything else like text or images

**Comment by Serene-Arc** [comment, marked as answer]
If you mean you only want to download the videos, you can use the `--skip` option. There isn't really another way to do that. If you want the links themselves, you can use the `archive` command and then sort through them with a parser for JSON or YAML.
